tensorflow/python/ops/image_ops_impl.py

```diff
@@ -395,6 +403,13 @@ def random_flip_left_right(image, seed=None):
   Returns:
     A tensor of the same type and shape as `image`.
 
+  Usage Example:
```

Comment:
Already an example is present in the function.

Reply:
didn't see that - will remove